sharding/collation_test.go

```diff
@@ -131,34 +132,153 @@ func makeTxWithGasLimit(gl uint64) *types.Transaction {
 // BENCHMARK TESTS
 
 // Helper function to generate test that completes round trip serialization tests for a specific number of transactions.
-func runBenchTest(b *testing.B, numTransactions int) {
+func makeRandomTransactions(numTransactions int) []*types.Transaction {
 	var txs []*types.Transaction
 	for i := 0; i < numTransactions; i++ {
 		data := make([]byte, 650)
```

Comment:
Tweaking this number affects the benchmark as well. I take it that the average ethereum transaction is around 650 bytes?

Reply:
I would say around 150 bytes per transaction going from the last few blocks in etherscan

Reply:
There is no basis for the number 650. I picked it at random.

Reply:
Gotcha. I'll change it to 150 bytes, since that's more in line with what's in the blockchain at the moment. That's also in line with the data that's being encoded in [this post](https://medium.com/@shijuvar/benchmarking-protocol-buffers-json-and-xml-in-go-57fa89b8525).